### PR TITLE
automating our environment by adding shell script and modifying vagrantfile

### PR DESCRIPTION
**Pull request by advapi**
- Setup Vagrant to automatically bring our application up as well as
make it accessible from a web browser
After running `vagrant up` the application can be accessed at:
> http://127.0.0.1:8000/
related issue: #2

### CODE REVIEW

File: bootstrap.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+# password if we don’t have sufficient privileges to run it
+sudo -n dnf install -y pipenv
+
+cd /vagrant
+# Install dependencies with Pipenv
+pipenv sync
+
+# Run database migrations
+pipenv run python manage.py migrate
+
```
Comment (bardielle) [inline bootstrap.sh]: Im not sure that this is necessary, 
Please check if you need it...

Comment (advapi) [reply bootstrap.sh]: the entire file or part of it? @bardielle

Comment (bardielle) [reply bootstrap.sh]: only line 14

Comment (advapi) [reply bootstrap.sh]: changed it @bardielle

Comment (bardielle) [reply bootstrap.sh]: :+1: 
Also remove line 13 (now it's not true that you're running DB migration)

Comment (advapi) [reply bootstrap.sh]: done :)

**Review by bardielle — APPROVED** [review]

**Pull request merged**